### DISCUSSION TITLE
FYI - if buttons seem to work intermittently...

### DISCUSSION THREAD

**Discussion by GTJester** [gcormier/megadesk#114] [General]
Try cleaning the ribbon connection. I was having trouble getting the up button to register clicks and I cleaned the ribbon with some 70% IPA that I had handy and it works beautifully now.

**Comment by ryanmatthews** [comment]
After extensive troubleshooting I had isolated the buttons as my problem. I assumed something inside the ribbon cable was shorted or damaged somehow. I found this post, cleaned the ribbon connector and bam - they work as reliably as the day I got it. Thanks for saving me time and money @GTJester.

**Comment by oldbean57** [comment]
Thank-you very much indeed for this advice @GTJester - this resolved just such an issue for me too. Working reliably again now.
